src/estimation/first_step_estimation/est_SRA_random_walk.py

```diff
@@ -30,6 +33,9 @@ def est_expected_SRA(paths, df):
 
 
 def estimate_expected_SRA_variance(paths, df):
+    # truncate data: remove birth years outside before 1964
+    df = df.query("birth_year >= 1964")
```

Comment:
You want to do this or should we create an issue?

Reply:
will take care when changing SRA process estimation method